### ISSUE TITLE
keyserver receive failed: No name

### ISSUE THREAD

**Issue by Billir** [tobyweston/temperature-machine#109]
Got this error when installing Temperature Machine 
Executing: /tmp/apt-key-gpghome.Hd8dHFNGHw/gpg.1.sh --keyserver pool.sks-keyservers.net --recv-keys 00258F48226612AE
gpg: keyserver receive failed: No name
Any suggestions??

**Comment by AlistairFielden** [comment]
I'm having the same issue.

**Comment by HenrysCat** [comment]
Just use Stretch 4.14, works every time, T.W. will probably not update this ;)

**Comment by Billir** [comment]
I found a workaround that is not all that satisfactory. If you make a clone
of your working model - In the rpi menu - accessories - cd card copier -
one can make a copy of the card that is running the pi. It works fine but
if you change the host name of the new device - it will read the
temperature of the sensor and display it through the web interface but will
not display the log graph. As I know very little about the logging and web
interface - I could not change it. If you just keep using the original host
name - you can put 2 devices on the same network. I just use the unique IP
address to select the device on the network but have to remember where each
is.

Bill Rayment

**Comment by DemBones78** [comment]
Honesty this is a dead program that is woafully out of date. The graphing
is terrible, it will only do Celsius. by default it aggregates all the
temperatures from your sensors.  Did I mention the graphing is terrible?
Lastly. TW I said I'm multiplications that this program was made for him
and he's not really interested in making improvements for anybody else.


Your best bet is to load hass.io on that raspberry pi and get an ESP32 or
espesp8266 .    Load up the esp with the esphome add on in hass.io and in a
few minutes you already , you will  have way more functionality than the
temperature machine. Also functionality is documented and customizable.
The default graphs in hassile are pretty good.

But if you set up InfluxDB and Grafana you will have best in class
completely customizable graphs.   All it will cost you is an esp  they are
3 for $10

At this point it's a snap to then further add all sorts of sensors such as
humidity sensors pressure sensors light level sensors whatever sensor
tickles your fancy most are readily available and compatible with ESP home.

And a major plus is all of this stuff is available via tutorials all over
forums and YouTube

**Comment by patrickn** [comment]
It seems that the SKS keyserver has been deprecated, all you need to do is use an alternative key server. This worked for me:
`   sudo apt-key adv --keyserver keyserver.ubuntu.com  --recv-keys 00258F48226612AE`

**Comment by Billir** [comment]
Thanks, I'll check it out.
Bill Rayment
Project Bee
1 403 630 4848

**Comment by Billir** [comment]
Well it worked using Stretch Version and
Patrick's

It seems that the SKS keyserver has been deprecated, all you need to do is
use an alternative key server. This worked for me:
sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys
00258F48226612AE

Got 3 sensors reading and graphing. It is shows only temperatures but no
graphs with Buster

Bill Rayment
Project Bee
1 403 630 4848

**Comment by Billir** [comment]
Well I was wrong - it did present a graph with the temp range but did not
log. Sad it did work well in the past.
Bill Rayment
Project Bee
1 403 630 4848

**Comment by DemBones78** [comment]
Yeah man just load home assistant on that raspberry pi you will be much
better off.   Hass.io

**Comment by Alireza1369newi** [comment]
> Got this error when installing Temperature Machine 
> Executing: /tmp/apt-key-gpghome.Hd8dHFNGHw/gpg.1.sh --keyserver pool.sks-keyservers.net --recv-keys 00258F48226612AE
> gpg: keyserver receive failed: No name
> Any suggestions??

**Comment by patrickn** [comment]
> > Got this error when installing Temperature Machine
> > Executing: /tmp/apt-key-gpghome.Hd8dHFNGHw/gpg.1.sh --keyserver pool.sks-keyservers.net --recv-keys 00258F48226612AE
> > gpg: keyserver receive failed: No name
> > Any suggestions??

Yes, SKS keyserver has been deprecated. See my reply above.

**Comment by Billir** [comment]
It is now April 2022 and this key provided earlier still works 
sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 00258F48226612AE

**Comment by iw2ejh** [comment]
sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 00258F48226612AE    is working now in decembre 2022, thanks
